llvm/lib/Target/AArch64/AArch64TargetTransformInfo.cpp

```diff
@@ -1160,6 +1160,10 @@ static std::optional<Instruction *> instCombineSVECmpNE(InstCombiner &IC,
                                                         IntrinsicInst &II) {
   LLVMContext &Ctx = II.getContext();
 
+  // Replace by zero constant when all lanes are inactive
+  if (auto II_NA = instCombineSVENoActiveUnaryZero(IC, II))
```

Comment:
Not relevant to this PR but `instCombineSVENoActiveUnaryZero` seems like a bad name given the context in which the function is now used. Do you mind committing a NFC rename patch to something like `instCombineSVENoActiveLanes`?

Reply:
Okay I will try coming up with better naming scheme for this. It cannot be just `instCombineSVENoActiveLanes ` as there are multiple different things that can happen when all lanes are inactive, but using it as a base can work.